cmd/deployment-check/README.md

```diff
@@ -74,7 +74,7 @@ spec:
   podSpec:
     containers:
       - name: deployment
-        image: kuberhealthy/deployment-check:v1.7.1
+        image: kuberhealthy/deployment-check:v1.8.2
```

Comment:
same with the versioning here -- too many jumps

Reply:
done